plugins/usbdmx/AsyncPluginImpl.cpp

```diff
@@ -175,6 +183,14 @@ bool AsyncPluginImpl::NewWidget(JaRuleWidget *widget) {
                                 new JaRuleDevice(m_plugin, widget, str.str()));
 }
 
+bool AsyncPluginImpl::NewWidget(NodleU1 *widget) {
+  return StartAndRegisterDevice(
+      widget,
+      new NodleU1Device(m_plugin, widget, "DMXControl Projects e.V. Nodle U1",
```

Comment:
Users may benefit from seeing the serial number in the title here, if patching multiple units in.